content/blogs.md

```diff
@@ -4,6 +4,20 @@ section: single
 type: page
 ---
 
+<h2><a href="https://kusari.dev/blog/guac-beta-announcement/">Quest to determine the 'G' in GUAC
```

Comment:
this has to be updated to the blog link, else lgtm

Reply:
fixed.